examples/step-89/doc/intro.dox

```diff
@@ -34,8 +34,29 @@ The set of equations consists of the conservation of mass and momentum
   \frac{\partial \, \mathbf{u}}{\partial \, t} + \frac{1}{\rho}\nabla p = \mathbf{0}.
 @f]
 Here, $p$ is the acoustic pressure, $\mathbf{u}$ the acoustic particle velocity, $c$ the
-speed of sound, and $\rho$ the mean density of the fluid in which waves are propagating.
-For the discretization we make use of discontinuous Galerkin (DG) methods. DG methods are
+speed of sound, and $\rho$ the mean density of the fluid in which waves are propagating. As stated
+above, the two equations are simply a different way of writing the wave equation: If
+you take the time derivative of the first equation, and the divergence of the second,
+i.e., compute
+@f[
+  \frac{\partial^2 \, p}{\partial \, t^2} + \rho c^2 \nabla\cdot \frac{\partial \mathbf{u}}{\partial t} = 0,\\
+  \frac{\partial \, \nabla \cdot \mathbf{u}}{\partial \, t} + \nabla \cdot \frac{1}{\rho}\nabla p = \mathbf{0},
+@f]
+then you can substitute the second equation into the first one to obtain
+@f[
+  \frac{\partial^2 \, p}{\partial \, t^2} - \rho c^2 \nabla \cdot \frac{1}{\rho}\nabla p = \mathbf{0},
+@f]
+which in the case of constant density $\rho$ results in the more familiar form of the wave
+equation:
+@f[
+  \frac{\partial^2 \, p}{\partial \, t^2} - c^2 \Delta p = \mathbf{0}.
+@f]
+
+The reason one may want to consider the original form is that it has the form of
+a *hyperbolic conservation law* in which only first time and spatial derivatives
+appear.
```

Comment:
Another motivation one might want to add here is that it is simpler to construct dissipative schemes in the first-order form: While the wave equation preserves the energy, any discretization scheme applied to it will suffer from some form of dispersion errors (albeit DG is doing pretty well, see the statement in the next paragraph). With DG and upwind-type fluxes, the scale range that is affected by dispersion errors is also dissipated, i.e., scales that cannot be transported accurately will get damped and eventually disappear. This can also be done for the wave equation in second-order form, but it requires considerably more advanced mathematics (I think one can apply spectral vanishing viscosity, but I am not an expert on that topic).

Reply:
I've updated this slightly, to the degree that I understand these issues. Feel free to expand on it in a follow-up patch since you know so much more about these issues than I do.